Function to create brain mask

jsjol opened the pull request:
Refers to #8 .

Basically a thin wrapper of dipy.segment.mask.median_otsu. Currently uses default settings, after #9 is resolved we should check if these settings work well for HCP data.

jsjol committed:
Created brain mask function

Basically a wrapper of dipy.segment.mask.median_otsu.

jsjol committed:
Brain mask now enforces positive signal

jsjol commented:
Is the test that the masked brain volume is of a certain size a good fit for a unit test? Is it more of a functional test? 

A problem is that it requires realistic data, which makes the test run in approx 15 s, which is way too long for a unit test. Still I think it is valuable to do a similar test somewhere.

marbank commented on test_createBrainMask.py:
```diff
@@ -0,0 +1,41 @@
+
+class TestCreateBrainMask(unittest.TestCase):
+
+    def test_createBrainMaskFromb0Data(self):
```
Add a function description to function and of function arguments.

mariabankestad commented on createBrainMask.py:
```diff
@@ -0,0 +1,12 @@
+
+def createBrainMaskFromb0Data(b0Data):
+    medianRadius = 4
+    numIterations = 4
```
Add a function description to function and of function arguments. Why MedianRadius = 4?

mariabankestad replied:
Added description in new commit. MedianRadius = 4 is the default, hence it unnecessary to define, so I removed it.

jsjol committed:
Reactions to code review, removed realistic test, added save functionality

Added description. Removed definition of parameter values to what was
anyway the default. Moved the test that brain volume is realistic to an
integration test instead. Added save functionality (shouldn't have been
part of this commit, sorry).

jsjol committed:
Merge remote-tracking branch 'refs/remotes/origin/master' into create-brain-mask

jsjol commented:
Closes #8
